commands/FBPrintCommands.py

```diff
@@ -299,3 +300,15 @@ def run(self, arguments, options):
       object = fb.evaluateObjectExpression(objectToMessage)
       printCommand = 'po [{} valueForKeyPath:@"{}"]'.format(object, keypath)
       lldb.debugger.HandleCommand(printCommand)
+
+
+class FBPrintApplicationDocumentsPath(fb.FBCommand):
+  def name(self):
+    return 'ppath'
```

Comment:
have thought about this before. It seems to me the word `docs path` is a little ambiguous, but use `ppath` here is just easy to type in. will change to the name suggested.